docs/guides/configure-error-suppression.mdx

```diff
@@ -1,12 +1,16 @@
 ---
-title: Configure runtime compilation
-description: How to configure runtime compilation in Qiskit Runtime.
+title: Configure error suppression
+description: How to configure error suppression in Qiskit Runtime. (Previous topic - runtime compilation)
```

Comment:
Is the parenthetical to make sure the keywords "runtime compilation" still show up in search? "Previous topic" sounds a bit like it could refer to the topic before this one. Maybe (Previously known as) or (Also known as) or something?

Reply:
I think so - Eric made that change.  Maybe "previously titled"?

Reply:
Former title